package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "pdp-checker",
+  "description": "A tool for checking PDP compliance and retrieval.",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "lint": "prettier --check . && standard",
+    "lint:fix": "prettier --write . && standard --fix .",
+    "test:types": "tsc -p .",
+    "test:unit": "node --test",
+    "test": "npm run lint && npm run test:types && npm run test:unit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/CheckerNetwork/pdp-checker.git"
+  },
+  "author": "Space Meridian <spark@meridian.space>",
```

Nitpick (feel free to ignore)

Given that this will be separate product from Spark we can use new email address related to this project.

```suggestion
  "author": "Space Meridian <pdp@meridian.space>",
```